src/vcode.h

```diff
@@ -105,6 +105,7 @@ typedef enum {
    VCODE_OP_COVER_STMT,
    VCODE_OP_COVER_BRANCH,
    VCODE_OP_COVER_TOGGLE,
+   VCODE_OP_COVER_EXPR,
```

Comment:
You need to increment `VCODE_VERSION` every time you make an incompatible change otherwise user will get confusing errors when loading existing files (adding an new opcode to the middle of the list changes all subsequent ones).